packages/redux-query-react/src/components/connect-request.js

```diff
@@ -54,6 +54,25 @@ const diffQueryConfigs = (
   return { cancelKeys, requestQueryConfigs };
 };
 
+export const headersChanged = (
+  queryConfigs: Array<QueryConfig>,
+  previousQueryConfigs: Array<QueryConfig>,
+) => {
+ return queryConfigs.map(function (config, i) {
+    if (previousQueryConfigs[i] && queryConfigs[i] && previousQueryConfigs[i].options && queryConfigs[i].options) {
+      var prevHeaders = previousQueryConfigs[i].options.headers;
```

Comment:
let/const should work here, right?

Reply:
yes, copy pasta from the already built file